bot/mconfig.py

```diff
@@ -0,0 +1,156 @@
+#
```

Comment:
This file should not be in this PR.

Reply:
@KKopyscinski could you please resolve this?